src/shared/components/palette/complex/read-complex-field-new-table.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit } from '@angular/core';
+import { AbstractFieldReadComponent } from '../base-field/abstract-field-read.component';
+
+@Component({
+  selector: 'ccd-read-complex-field-new-table',
+  templateUrl: './read-complex-field-new-table.html',
+  styleUrls: ['./read-complex-field-new-table.scss']
+})
+export class ReadComplexFieldNewTableComponent extends AbstractFieldReadComponent implements OnInit {
+
+  public isDisplayContextParameterAvailable = false;
+  public columns: String[];
+  public columnsLabel: String[];
+  public columnsVerticalLabel: any;
+  public columnsHorizontalLabel: any;
+  public columnsHorizontalLabelVisibility: any;
+  public columnsAllLabels: any;
+
+  public isHidden: boolean[] = [];
+
+  ngOnInit(): void {
+    if (this.caseField.display_context_parameter && this.caseField.display_context_parameter.trim().startsWith('#TABLE(')) {
+      this.isDisplayContextParameterAvailable = true;
+      let displayContextParamter = this.caseField.display_context_parameter.trim();
+      let result: string = displayContextParamter.replace('#TABLE(', '');
+      this.columns = result.replace(')', '').split(',');
+      let labels = '';
+      let labelsVertical: { [k: string]: any } = {};
+      let labelsHorizontal: { [k: string]: any } = {};
+      let labelsHorizontalVisibility: { [k: string]: any } = {};
+      let allLabels: { [k: string]: any } = {};
+      for (let obj of this.caseField.value) {
```

Comment:
Why a loop here if the block does not use obj?

Reply:
Ok you need that array populated.

Reply:
Yes, so I can track the order of the show and hide objects.